src/system_metrics/service_mongoose_system_metrics.erl

```diff
@@ -0,0 +1,67 @@
+-module(service_mongoose_system_metrics).
+-author('jan.ciesla@erlang-solutions.com').
+
+-behaviour(gen_server).
+
+-define(DEFAULT_INITIAL_REPORT, 5 * 60 * 1000).     % 5 min intial report
```

Comment:
What do you think about changing the unit here (and in configuration) to minutes and do the calculation when we read the values?

Reply:
Milliseconds are universal format used everywhere, I'm afraid that someone will just assume it's milliseconds (either in here or in the cfg files) and be surprised with the delay. In the macro I wanted to have it readable but still in milliseconds.